local/kcp/start.sh

```diff
@@ -123,6 +121,8 @@ if [[ -z "${PARAMS}" ]]; then
 fi
 
 kcp-start
+printf "KUBECONFIG=%s\n" "${KUBECONFIG}"
+printf "kubectl kcp plugin: %s\n" "${KCP_DIR}/bin/kubectl-kcp"
```

Comment:
The standard way of using a kubectl plugin is to have it copied to the same location as the kubectl binary and then calling the kubectl binary rather than directly the plugin.
I think that we should instruct the user to copy the plugin binary to the same location as the kubectl binary.

Reply:
I have [updated](https://github.com/openshift-pipelines/pipelines-service/compare/43cdfd1c247b5538b68ab828a8695a6d13081eb6..774612422644696194afaa69b56b04cd43997ce8) the output message to reflect that.
Let me know if that's ok or we need to provide more information